corehq/apps/app_manager/models.py

```diff
@@ -1305,6 +1305,7 @@ class Form(IndexedFormBase, NavMenuItemMediaMixin):
     form_filter = StringProperty()
     requires = StringProperty(choices=["case", "referral", "none"], default="none")
     actions = SchemaProperty(FormActions)
+    case_references_data = DictProperty()
```

Comment:
I like the idea of having a generic case references property, but I think it could cause some confusion when saving is implemented as this is actually specific to loading